rosetta/admin.py

```diff
@@ -0,0 +1,63 @@
+from functools import update_wrapper
+
+from django.contrib import admin
+
+from rosetta.signals import reload_server
+
+try:
+    from django.urls import reverse_lazy
+except:
+    from django.core.urlresolvers import reverse_lazy
+from django.views.generic import RedirectView
+
+from rosetta.models import TranslationBackup, Translation
+from rosetta.conf import settings as rosetta_settings
+
+
+def restore(modeladmin, request, queryset):
+    for backup in queryset:
+        backup.restore()
+
+    if rosetta_settings.AUTO_RELOAD:
+        reload_server.send(sender=None, request=request)
+
+
+class TranslationBackupAdmin(admin.ModelAdmin):
+    actions_on_bottom = True
+    actions = [restore]
+    save_on_top = True
+    fields = ('created', 'locale_path', 'language', 'domain', 'content')
+    list_display = ('created', 'locale_path', 'language', 'domain')
+    list_filter = ('created', 'language', 'domain')
+    readonly_fields = ('created', 'locale_path', 'language', 'domain')
+
+    def has_add_permission(self, request):
+        return False
+
+
+class TranslationsAdmin(admin.ModelAdmin):
+    def get_urls(self):
+        from django.conf.urls import url
+
+        def wrap(view):
```

Comment:
Maybe it can work with just extending RedirectView, and not with monkey patching.